Makefile

```diff
@@ -11,12 +11,12 @@ MODE ?= set
 
 # for dockerized prototool
 USER := $(shell id -u):$(shell id -g)
-DOCKER_BASE := docker run --rm --user $(USER) --mount type=bind,source="$(shell pwd)",target=/work --tmpfs /tmp:exec iov1/prototool:v0.2.0
+DOCKER_BASE := docker run --rm -v $(shell pwd):/work iov1/prototool:v0.2.0
```

Comment:
Nice simplification. I copied the old line.

If you don't reference the USER variable anywhere, you can remove that line